pensjon-brevbaker/src/test/kotlin/no/nav/pensjon/brev/maler/adhoc/AdhocTest.kt

```diff
@@ -0,0 +1,47 @@
+package no.nav.pensjon.brev.maler.adhoc
+
+import kotlinx.coroutines.runBlocking
+import no.nav.pensjon.brev.*
+import no.nav.pensjon.brev.latex.LaTeXCompilerService
+import no.nav.pensjon.brev.template.Language
+import no.nav.pensjon.brev.template.Letter
+import no.nav.pensjon.brev.template.LetterTemplate
+import no.nav.pensjon.brev.template.render.PensjonHTMLRenderer
+import no.nav.pensjon.brev.template.render.PensjonLatexRenderer
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+
+@Tag(TestTags.PDF_BYGGER)
+class AdhocTest {
+    fun testHtml(template: LetterTemplate<*, *>, htmlName: String) {
+        Letter(template, Unit, Language.Bokmal, Fixtures.fellesAuto)
+            .let { PensjonHTMLRenderer.render(it) }
+            .also { writeTestHTML(htmlName, it) }
+    }
+
+    fun testAdhocPdf(template: LetterTemplate<*, *>, pdfName: String) {
+        Letter(template, Any(), Language.Bokmal, Fixtures.fellesAuto)
```

Comment:
Hvorfor bruker du `Any()` her i stedet for `Unit?`

Reply:
Vet ikke. Endret.